microceph/ceph/ceph_rbd_mirror.go

```diff
@@ -0,0 +1,24 @@
+package ceph
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+func bootstrapRbdMirror(hostname string, path string) error {
+	args := []string{
+		"auth",
+		"get-or-create",
+		fmt.Sprintf("client.rbd-mirror.%s", hostname),
+		"mon", "profile rbd-mirror",
+		"osd", "profile rbd",
+		"-o", filepath.Join(path, "keyring"),
+	}
+
+	_, err := cephRun(args...)
+	if err != nil {
```

Comment:
Nit: the if... return seems unnecessary here

Reply:
added error logging in the branch.